app/views/mailers/notifications_mailer/onboarding_product.html.haml

```diff
@@ -0,0 +1,25 @@
+%p Hey there,
+
+%p
+  It's been a day or so since you signed up to #{link_to 'Exercism', 'https://exercism.org'}, and I wanted to see how you're getting on so far?
```

Comment:
Maybe?
```suggestion
  It's been a day or so since you signed up to #{link_to 'Exercism', 'https://exercism.org'}, and I wanted to check how you're getting on so far?
```